### PR TITLE
add DB_HOST to link with mysql host

### PR DESCRIPTION
**Pull request by AlvaroVega**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -24,19 +32,20 @@ RUN mvn package
...
 RUN sed -i "s/port: 8081/port: 7071/g" /opt/keypass/config.yml
+RUN sed -i "s/mysql:\/\/localhost/mysql:\/\/"$DB_HOST"/g" /opt/keypass/config.yml
 
 WORKDIR /opt/keypass
 
-RUN java -jar keypass-0.4.4.jar db migrate config.yml
-
-CMD ["java -jar keypass-0.4.4.jar server config.yml"]
+# Define the entry point
+ENTRYPOINT ["/opt/keypass/cygnus-entrypoint.sh"]
```
Comment (dmoranj) [inline Dockerfile]: I assume `cygnus-entrypoint.sh` here is an error...

File: Dockerfile [hunk 2]
```diff
@@ -2,41 +2,53 @@ FROM centos:6
...
 
-RUN mkdir -p /opt/keypass
-RUN mkdir -p /opt/keypass/log
-COPY ./target/keypass-*.jar /opt/keypass/
+RUN \
+    mvn package && \
+    mkdir -p /opt/keypass && \
+    mkdir -p /opt/keypass/log
+
+COPY ./target/keypass-$KEYPASS_VERSION.jar /opt/keypass/keypass.jar
```
Comment (dmoranj) [inline Dockerfile]: Do this line copy the file generated in line 33 using `mvn package` or does it copy the .jar in the local folder (outside the Docker image)?

Comment (AlvaroVega) [inline Dockerfile]: This copy the jar generated by `mvn package` into /opt/keypass, changing the name of the jar

File: Dockerfile [hunk 3]
```diff
@@ -2,41 +2,53 @@ FROM centos:6
...
 
 
-RUN sed -i "s/port: 8080/port: 7070/g" /opt/keypass/config.yml
-RUN sed -i "s/port: 8081/port: 7071/g" /opt/keypass/config.yml
+RUN \
+    sed -i "s/port: 8080/port: 7070/g" /opt/keypass/config.yml && \
+    sed -i "s/port: 8081/port: 7071/g" /opt/keypass/config.yml && \
+    sed -i "s/mysql:\/\/localhost/mysql:\/\/"$DB_HOST"/g" /opt/keypass/config.yml
 
 WORKDIR /opt/keypass
```
Comment (dmoranj) [inline Dockerfile]: Redundant, already on line 30

File: Dockerfile [hunk 4]
```diff
@@ -2,41 +2,53 @@ FROM centos:6
...
-## Install MySQL (client at least?)
-RUN yum -y install mysql
+RUN \
+    # Install dependencies
+    yum update -y && yum install -y wget  && \
+    wget http://dl.fedoraproject.org/pub/epel/6/x86_64/epel-release-6-8.noarch.rpm && \
+    yum localinstall -y --nogpgcheck epel-release-6-8.noarch.rpm  && \
+    yum install -y rpm-build git java-1.7.0-openjdk java-1.7.0-openjdk-devel
+
+ENV JAVA_HOME /usr/lib/jvm/java-1.7.0-openjdk.x86_64
```
Comment (dmoranj) [inline Dockerfile]: You may move this ENV sentence above the previous RUN or below the next, so you can use a single RUN sentence for both (neither of them use Java, so it doesn't seem important to define it in the middle).

**Comment by dmoranj** [comment]
LGTM

**Pull request merged**